playbooks/roles/maas_entities/tasks/main.yml

```diff
@@ -6,7 +6,7 @@
       api_key: "{{ lookup('env', 'PUBCLOUD_API_KEY') }}"
       state: "{{ state }}"
       label: "{{ item }}"
-      region: "{{ lookup('env', 'REGION') }}"
+      region: "{{ lookup('env', 'REGIONS') }}"
```

Comment:
What happens if `REGIONS` is a list or a the region used was taken from `FALLBACK_REGIONS`?

Reply:
Addressed this and save_pubcloud.yml by using rax_region from the generated static inventory.